main.go

```diff
@@ -567,6 +580,26 @@ func writeSchemaFile(conv *internal.Conv, now time.Time, name string, out *os.Fi
 	fmt.Fprintf(out, "Wrote schema to file '%s'.\n", name)
 }
 
+func writeSessionFile(conv *internal.Conv, now time.Time, name string, out *os.File) {
+	f, err := os.Create(name)
+	if err != nil {
+		fmt.Fprintf(out, "Can't create session file %s: %v\n", name, err)
+		return
+	}
+	// Session file will basically contain 'conv' struct in JSON format.
+	// It contains all the information for schema and data conversion state.
+	convJSON, err := json.MarshalIndent(conv, "", " ")
```

Comment:
conv has a lot of stuff.
Most of it is small.
But sampleBadRows could be fairly big (we have a hard-coded 10MB limit).

Maybe worth a comment in the conv definition that this datastructure is serialized into session.json, so care should be take adding large new datastructures into conv?
We could also consider dropping some parts of conv.

Reply:
As noted during the discussion earlier today, we write out conv before we do data conversion.
So sampleBadRows will always be empty.
Also, sampleBadRows is lower case, so it won't be written to JSON.